deposit-app/src/test/java/edu/unc/lib/boxc/deposit/validate/ValidateContentModelJobTest.java

```diff
@@ -211,6 +213,75 @@ public void missingOriginalDatastreamTest() {
         });
     }
 
+    @Test
+    public void fileObjectNoOriginalDatastreamWithAllStreamingPropertiesTest() {
+        PID objPid = makePid(CONTENT_BASE);
+        Bag objBag = model.createBag(objPid.getRepositoryPath());
+        objBag.addProperty(RDF.type, Cdr.Work);
+
+        PID childPid = makePid(CONTENT_BASE);
+        Resource childResc = model.getResource(childPid.getRepositoryPath());
+        childResc.addProperty(RDF.type, Cdr.FileObject);
+        childResc.addProperty(Cdr.streamingFile, "banjo-playlist.m3u8");
+        childResc.addProperty(Cdr.streamingFolder, CLOSED);
+        childResc.addProperty(Cdr.streamingHost, DURACLOUD);
+        objBag.add(childResc);
+
+        objBag.addProperty(Cdr.primaryObject, childResc);
+
+        depBag.add(objBag);
+
+        job.closeModel();
+
+        job.run();
+    }
+
+    @Test
+    public void fileObjectNoOriginalDatastreamWithOneStreamingPropertyTest() {
+        Assertions.assertThrows(JobFailedException.class, () -> {
+            PID objPid = makePid(CONTENT_BASE);
+            Bag objBag = model.createBag(objPid.getRepositoryPath());
+            objBag.addProperty(RDF.type, Cdr.Work);
+
+            PID childPid = makePid(CONTENT_BASE);
+            Resource childResc = model.getResource(childPid.getRepositoryPath());
+            childResc.addProperty(RDF.type, Cdr.FileObject);
+            childResc.addProperty(Cdr.streamingFile, "banjo-playlist.m3u8");
+
+            objBag.add(childResc);
+
+            objBag.addProperty(Cdr.primaryObject, childResc);
+
+            depBag.add(objBag);
+
+            job.closeModel();
+
+            job.run();
+        });
+    }
+
```

Comment:
I'd suggest adding a test for a fileObject with all streaming properties and a valid original datastream. Maybe also one for a file object with all streaming properties plus an original datastream but no storage location.